build/steps.yml

```diff
@@ -5,18 +5,18 @@ steps:
 - template: steps-init.yml
 
 
-- powershell: ./build.ps1
+- pwsh: ./build.ps1
```

Comment:
for my education: what's the difference and why to use pwsh over powershell?

Reply:
Up to and including version 5.1, PowerShell was based on .NET Framework and called "Windows PowerShell." With version 6.0, they renamed to "PowerShell Core" and based on .NET Core instead. With version 7.0, they renamed again to "PowerShell" with no other qualifications, mirroring the ".NET Framework" → ".NET Core" → ".NET 5" naming path. In Azure Pipelines, the `powershell` task gives PowerShell 5.1 on Windows agents, even though that version is now long deprecated and has some compatibility bugs that impede cross-platform support. Whenever possible, using `pwsh` is suggested to ensure that 6.0 or later is used.

Reply:
Thanks for the explanations, Chris!

Reply:
No worries, happy to help!